### PR TITLE
reviewing the model and tests ref #89 and ref #87

### PR DESCRIPTION
**Pull request by fabianazioti**

### CODE REVIEW

File: examples/create_class_system.py [hunk 1]
```diff
@@ -31,40 +32,42 @@
...
     }
 ]
 
-def verify_class_system_exist(class_system_name):
 
+def verify_class_system_exist(class_system_name):
     try:
-        class_system = LucClassificationSystem.get(name=class_system_name)
-        return class_system
+        classificationn_system = LucClassificationSystem.get(name=class_system_name)
```
Comment (M3nin0) [inline examples/create_class_system.py:45]: I think here you have an extra "n"

File: lccs_db/utils.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#
+# Land Cover Classification System Database Model. is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+#
+
+"""Utility for Land Cover Classification System Database Model."""
+import mimetypes
+
+
+def validate_mimetypes(style_file):
```
Comment (M3nin0) [inline lccs_db/utils.py:13]: I think this method may have a name that refers to the type of recovery. For example, get_mimetype or something like that. What do you think?

**Review by raphaelrpl — APPROVED** [review]

File: tests/test_ext.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#
+# This file is part of BDC-Catalog.
```
Comment (raphaelrpl) [inline tests/test_ext.py:2]: Did you mean LCCS-DB?

File: tests/test_ext.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+# Copyright (C) 2019-2020 INPE.
+#
+# BDC-Catalog is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+#
+
+"""Unit-test for extension BDC-Catalog."""
```
Comment (raphaelrpl) [inline tests/test_ext.py:9]: Same here

**Review by raphaelrpl — CHANGES_REQUESTED** [review]

**Review by M3nin0 — APPROVED** [review]

File: lccs_db/__init__.py [hunk 5]
```diff
@@ -7,8 +7,14 @@
...
 """Python Land Cover Classification System Database Model."""
+import mimetypes
 
 from .ext import LCCSDatabase
 from .version import __version__
 
-__all__ = ('__version__', 'LCCSDatabase',)
\ No newline at end of file
+mimetypes.init()
+mimetypes.add_type("application/qml ", ".qml", True)
```
Comment (gqueiroz) [inline lccs_db/__init__.py:16]: Check if the is a standard for QGIS QML files, otherwise use `application/xml`.

File: lccs_db/__init__.py [hunk 6]
```diff
@@ -7,8 +7,14 @@
...
+mimetypes.add_type("application/sdl ", ".sdl", True)
```
Comment (gqueiroz) [inline lccs_db/__init__.py:17]: Take a look at the OGC recommendation below:
- https://github.com/opengeospatial/ogcapi-styles/blob/master/standard/clause_7_styles-api.adoc#requirements-class-ogc-sld-10

GeoServer doesn't seem to use these mimetypes:
- https://github.com/geoserver/geoserver/blob/e893e997434f0ca1b085896e27b1d774f8cafb6a/src/main/src/main/java/org/geoserver/catalog/SLDHandler.java#L66

My recommendation is to follow the OGC Styles-API in this case.

**Review by gqueiroz — CHANGES_REQUESTED** [review]

File: lccs_db/__init__.py (hunk 5 above)
Comment (fabianazioti) [reply lccs_db/__init__.py]: We didn't find a standard for the qml format

**Review by gqueiroz — APPROVED** [review]

**Pull request merged**